lib/src/core/toastification_manager.dart

```diff
@@ -60,10 +63,12 @@ class ToastificationManager {
 
       delay = const Duration(milliseconds: 300);
```

Comment:
@payam-zahedi Why the 300 millisecond delay? I've seen users complaining about it, and I've also noticed that the first toast is shown with a delay.